images/jenkinsoperator/Dockerfile

```diff
@@ -17,9 +17,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     pkg-config \
     procps \
     python \
+    python3 \
     python-dev \
     python-openssl \
     python-pip \
+    python3-pip \
```

Comment:
Were you able to build this image locally? When I tried I needed to add the pip3 set up tools.

Reply:
python3-pip installs fine for me. However, the pip3 update failed and that might've been what you were seeing. I've added changes to the upgrade line below to fix it.